mix_tasks/update_geoname_data.ex

```diff
@@ -30,6 +30,8 @@ defmodule Mix.Tasks.UpdateGeonameData do
     IO.puts("Writing result to #{@allcountries_dest}")
 
     File.write!(@allcountries_dest, Enum.join(result, "\n"))
+
+    Location.Scraper.write_date_to_version()
```

Comment:
Can't comment on non-diff lines, that's a new one 🤔. This module reports warnings about `@allcountries_src` set but never used. Turn it into a comment by the way, in case the URL is somehow good to keep around?

Reply:
FWIW I planned to handle it in #19